### PR TITLE
initial commit for CloudWatch log group encrypted rule

### PR DESCRIPTION
**Pull request by rafihm**
I confirm these files are made available under CC0 1.0 Universal (https://creativecommons.org/publicdomain/zero/1.0/legalcode)

*Issue #, if available:*
#233 
*Description of changes:*
Initial commit.

### CODE REVIEW

**Comment by rafihm** [comment]
@jongogogo Please review the PR. Thanks

File: python/CLOUDWATCH_LOG_GROUP_ENCRYPTED/CLOUDWATCH_LOG_GROUP_ENCRYPTED_test.py [hunk 1]
```diff
@@ -0,0 +1,247 @@
...
+    rule_invalid_parameter_value = '{"KmsKeyId": "asdfa97asf8a0sf09sa8df0a98sd0f8as0f8d0"}'
+    rule_empty_parameter = '{}'
+
+    zero_log_groups = {
+        "logGroups": [
+        ]
+    }
+
+    log_group_encrypted = {
+        "logGroups": [
```
Comment (jongogogo) [inline python/CLOUDWATCH_LOG_GROUP_ENCRYPTED/CLOUDWATCH_LOG_GROUP_ENCRYPTED_test.py]: Remove all the data which is not used.

File: python/CLOUDWATCH_LOG_GROUP_ENCRYPTED/CLOUDWATCH_LOG_GROUP_ENCRYPTED.py [hunk 2]
```diff
@@ -0,0 +1,445 @@
...
+    return all_log_groups
+
+def evaluate_parameters(rule_parameters):
+    if 'KmsKeyId' not in rule_parameters:
+        return {}
+
+    if 'arn:aws:kms' not in rule_parameters['KmsKeyId']:
+        raise ValueError('Invalid value for paramter KmsKeyId, Expected KMS Key ARN')
+
+    return rule_parameters['KmsKeyId']
```
Comment (jongogogo) [inline python/CLOUDWATCH_LOG_GROUP_ENCRYPTED/CLOUDWATCH_LOG_GROUP_ENCRYPTED.py]: Please pass the dictionnary. We try to keep the type the same across all rules.

**Comment by rafihm** [comment]
Thanks for the review @jongogogo I have done the requested modifications.

**Pull request merged**